public/serverScripts.js

```diff
@@ -58,3 +58,17 @@ async function calc() {
   document.getElementById('answer').innerHTML = `= ${json.answer}`;
   document.getElementById('resultInput').value = '';
 }
+
+document.addEventListener('DOMContentLoaded', function(){
```

Comment:
from linter, standard prefer using arrow function notation instead of function without name

Reply:
can you give an example, i dont get it

Reply:
instead of having
`function(input) {...}`

you would have 
`(input) => {...}`

since they dont need names anyways as anonymous functions